chore: update common_design

lazysoundsystem opened the pull request:
Refs: RW-835

There are more changes to be made, not least concatenating the css files in _includes/head.html , but this is much closer to the CD now.

For testing, either:
* set up Jekyll locally https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/creating-a-github-pages-site-with-jekyll and run `jekyll serve` from within the top level of the repo
* unpack the zipped archive `tar -xf built-site.tar.gz`, cd into the resulting `127.0.0.1:4000` directory and serve from there, with, for example, `php -S localhost:8002`. Then visit `localhost:8002` in the browser.

The main changes are the typography, layout, table and toc components. 

As mentioned above, the CSS should probably be concatenated into one file, but thought it might be easier to review/ improve while the css files are separate.

lazysoundsystem committed:
chore: update common_design

Refs: RW-835

lazysoundsystem committed:
chore: fix internal links

Refs: RW-835

lazysoundsystem committed:
chore: update colors and fix paths

Refs: RW-835

lazysoundsystem committed:
chore: add README and a zip of the static site for testing locally
without jekyll

Refs: RW-835

lazysoundsystem committed:
chore: update url

Refs: RW-835

lazysoundsystem committed:
chore: use .int for response

Refs: RW-835

lazysoundsystem committed:
chore: include normalize css and fix menu

Refs: RW-835

left23 reviewed (approved):
Looks good, approving.

lazysoundsystem committed:
chore: concatenate css

Refs: RW-835

lazysoundsystem committed:
fix: make the menu dropdown work

Refs: RW-835

Note that this is using some css to do what is handled by js in other
common design sites. The toggling needs fixing for consistency.
